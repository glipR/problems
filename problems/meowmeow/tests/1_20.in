91
bqvotoghisssssssssssssssssssssssssssssssssssssssssssssss
tril
zprsgftriiiiiiiiiiiiiiiiiiiiiiiiiilllllllllllllllll
hissssssssu
v
hissssssssssssssssssssssssssssssssssssssss
triiiiiiiiiiiiiiiiiiiiillllllllllllll
ufuk
kvewpxyyqovrzesc
triiiiiiiiiiiiiiillllllllllllllll
fxzoxbjxpjptvnmxuv
hisss
hisssssssssssssssssssssssss
hissssssssssssssssssssssssssssssssssssssssssss
vnapwel
kafsjqwhisssssssssssssssssssssssssssssss
eegzsfbrblle
yfcnz
bqoujnhzr
myqytztril
kccieimlukyzvgtetuc
jpwrffufjyz
fui
usxkwgkz
cqrxvvhissss
snqehisssssssssssssssssssssssssssssssss
dsbzettriiiiiiiiiiiiiiiiiiiiiiillllllllllllllllllllllllv
hghaqqohissssssssssssssssssssssssssssssssssss
hissssssssss
buuuuuuuuuuuuuuuuuuuuuuuurrrrrrrrrrblle
triiiiilllllllllllllllllllx
hissssssssssssssssssssssss
nnkfjpinhisxssssssssssssssssssssssss
buuuuuuuuuuuuuuuuuuuuuuuuuuurrrrrrrrrrrrrrrrrrbllllllllllllllllllle
hissssstsssssssssssssssssssss
triiiiiiiiiiiiiiiiiiiiiiilllllllllllllllllllllllllllll
buuurrrrrrrrrrrrrrrblllllled
tnmhissssssssssssssssssssssssssssssssssssssss
mbyqcqaksfwhuwxtuhx
hissss
hisssssssssssssssssss
his
tisssssssssssssssssssssss
juuuuuuuuurrrrrrrrrrrrrrrrrrrrrrrrrblle
hss
trll
buuuuuuuuuuuuuuurblllllllllllllllllllllllle
bngeckxmp
ixx
triiiiiiiiiiiiiiiiiiiiiiiiiiiiilllllllllllllllllllllllllll
hiscs
zv
buuuuguuuuurrrrrrrrrrrrrrrrrrrrrrrrbllllllllllle
buuuuuuuuurrrerrrrrrrrbllllllllllllle
burbe
ptdaxvt
his
esnazbrblle
hissssssssssssssssssssssssssssssssssssssssssv
triiiiiiiiiiillllllllllllllllllllllll
his
zkizjxxrhissssssssssssssssssssssssssssssssssssssssssssssss
hisssssssssssssu
hisssssssssssssssssssssssssssssssssssssssssss
hisssssssssssssssssssssssssssssssssssssss
fzuhahkrmnburbe
treiiill
uiubbaiqu
buuuuwuuuuuuuurrrrrrrrrrrrrrrrrrrrrrrrrrrblllllllllllllllllllle
ycebfhwvs
triiiziiiiiiiiiiiilllll
triiiiiiiiiiiiiiiiiiiiiilllllllllllll
hisssssssssssssssssw
hissssmsssssssssssssssssssssssssss
hissssssss
triiiniiiiiiiiiillllllllllllllll
hissssssssssssssssssssssssssssss
triiiiiiiiiiiiiimiiiiiiiillllllllllllllll
mqhwpswhxubuuuuuurrrrrrrrrrrrrrrrrrrrrrrrbllle
triiiiiiiiiiiiiiiiiiiiiiiiiiillllllllllllf
moljgnlxbku
triilllllllllllllllllllllll
hisssssssssssssssssssssssssssssssssssssssr
mmjtblxxhadr
hisssssssssssssssssssssssmsssssssssssssss
triiiiiiiiiiiiiiiiiiilllllllllllllllla
thltounwechgqzurgtuh
wouro
hissssssss
hisssssssssssssssssssssssssssssn
hissssssssssssssssssssso