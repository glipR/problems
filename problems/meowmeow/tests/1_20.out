human noises
human noises
human noises
human noises
human noises
hiss
trill
human noises
human noises
trill
human noises
hiss
hiss
hiss
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
hiss
burble
human noises
hiss
human noises
burble
human noises
trill
human noises
human noises
human noises
hiss
hiss
human noises
human noises
human noises
human noises
human noises
burble
human noises
human noises
trill
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
human noises
trill
human noises
human noises
human noises
hiss
hiss
human noises
human noises
human noises
human noises
human noises
human noises
trill
human noises
human noises
hiss
human noises
hiss
human noises
human noises
human noises
human noises
trill
human noises
human noises
human noises
human noises
human noises
human noises
hiss
human noises
human noises
